### ISSUE TITLE
realm total is wrong

### ISSUE THREAD

**Issue by Daeveren** [Septh/WoW-Broker_Cash#4]
![2018-07-21_16-13-53](https://user-images.githubusercontent.com/6321337/43035996-33a313fc-8d01-11e8-895c-ab0a079195d8.png)

**Comment by Septh** [comment]
Could you please elaborate? A simple screenshot won't help me identifying the problem here.
- are you a new user or did you upgrade from a previous version?
- did you faction/realm/name change your Priesto toon?
- do you have other toons on other realms and are the total for these realms wrong also?
I'll be happy to help you, but you have to help me first. ;)

**Comment by Daeveren** [comment]
Sure. New user. This happened once each new char was being added to the database. I spotted it first after logging my 2nd character, Priesto - as can be seen from the photo, Priesto's gold has been added to the list, it's just not counted in the Total. After I've logged a 3rd char, the total for the first 2 chars is correctly being shown, but 3rd is missing. That's all there is.
I don't have the addon anymore, deleted it as it's not what I'm looking for.
Hope this helps you figure what was wrong there.
Cheers!
